docs/addons/addon-types.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Types of addons'
+---
+
+Each Storybook addon is classified into two general categories, UI-based or Presets. Documented here are some of the features available for each type of addon. Use it as a reference when creating your next Storybook addon.
+
+## UI-based addons
+
+This particular type of addons allows you to customize Storybook's UI. If the addon you're developing falls into this category you can use some of the following elements to further customize the interface.
+
+### Panels
```

Comment:
Could you add images of these UI elements from @domyen's post? https://storybook.js.org/blog/addon-catalog-sneak-peak/

Reply:
@winkerVSbecks probably better to use an image that matches the code snippet for accuracy. I'm going to commit them shortly.

Reply:
Images are added with the latest commits.